linux/debian/libkeymancore.symbols

```diff
@@ -15,6 +15,7 @@ libkeymancore.so.1 libkeymancore #MINVER#
  km_core_context_length@Base 17.0.195
  km_core_context_set@Base 17.0.195
  km_core_context_shrink@Base 17.0.195
+ km_core_cp_dispose@Base 17.0.241
```

Comment:
You should use the version number that is current when it gets merged otherwise the API check will flag it.